reana/reana_dev/git.py

```diff
@@ -991,4 +992,46 @@ def _git_create_pr(comp):
             _git_create_pr(component)
 
 
+@click.option(
+    "--component",
+    "-c",
+    required=True,
+    multiple=True,
+    help="Which components? [name|CLUSTER]",
+)
+@git_commands.command(name="git-tag")
+def git_tag(component):  # noqa: D301
+    """Create the corresponding git tag for components with release commits.
+
+    \b
+    :param components: The option ``component`` can be repeated. The value may
+                       consist of:
+                         * (1) standard component name such as
+                               'reana-workflow-controller';
+                         * (2) short component name such as 'r-w-controller';
+                         * (3) special value '.' indicating component of the
+                               current working directory;
+                         * (4) special value 'CLUSTER' that will expand to
+                               cover all REANA cluster components [default];
```

Comment:
`[default]` should not be here, we should require a component to be passed, as in the other PR.